README.md

```diff
@@ -1 +1,89 @@
-# path-18-app
\ No newline at end of file
+# path-18-app
+
+Designed to give young adults in Israel a greater exposure to a wide variety of career paths. We also show users specific Universities they can apply to with their current grades.
+
+### Contributors
+
+[@veredrec](https://github.com/veredrec)
+
+[@matthewdking](https://github.com/matthewdking)
+
+### Problem Statement:
+* Young people have little exposure to the variety of jobs and careers paths they could pursue.
+* Uncertainty around career prospects.
+* Little knowledge of the path to take to get a specific job.
+
+### Run the app
+
+Clone the repository
+```
+git clone https://github.com/facn2/path-18-app.git
+```
+Install packages
+```
+npm install
+```
+Start app
+```
+npm start
+```
+Run tests
+```
+npm test
+```
+
+### Technologies
+
++ [create-react-app](https://github.com/facebookincubator/create-react-app)
++ [redux](https://redux.js.org/)
++ [PostgreSQL](https://www.postgresql.org/)
+
+### Database schemas
+
+**Users**
+
+| user_id | first_name | last_name  | email | grade_bagrut | grade_psychometric | grade_tawjihi |
+| ------- |:----------:|:----------:|:-----:|:------------:|:------------------:|:-------:|
+| 1       |            |            |       |              |                    |         |
+| 2       |            |            |       |              |                    |         |
```

Comment:
I'm not sure about the blank examples here. Just one line with examples or even just the titles without examples...